src/linear_solver_logs/solve_log_ma.jl

```diff
@@ -25,21 +36,28 @@ mutable struct MAInfo
 end
 
 """ 
-    DistInfo
+    SEDistInfo
 
 A mutable structure that stores information about the sub-Exponential distribution.
 
 # Fields
-- `sampler::Union{DataType, Nothing}`, The type of sampler being used.
-- `max_dimension::Int64`, The dimension that is being sampled.
-- `block_dimension::Int64`, The sampling dimension.
-- `sigma2::Union{Float64, Nothing}`, The variance parameter in the sub-Exponential distribution, 
+- `sampler::Union{DataType, Nothing}`, the type of sampler being used.
+- `max_dimension::Int64`, the dimension that is being sampled.
+- `block_dimension::Int64`, the sampling dimension.
+- `sigma2::Union{Float64, Nothing}`, the variance parameter in the sub-Exponential distribution, 
```

Comment:
The descriptions need more elaboration. You also need a reference in one of these.

Reply:
I am unsure if you were referring to a paper reference which has been included.